content/_data/changelogs/weekly.yml

```diff
@@ -1289,6 +1289,69 @@
     # pull: 3055 too minor (test fix)
     # pull: 3063 too minor (test fix)
     # pull: 3070 too minor (suppress test)
+- version: "2.86"
+  date: 2017-10-22
+  changes:
+    - type: major rfe
+      pull: 3076
+      message: >
+        <em>Launch agent via execution of command on the master</em> has been moved to a new <em>Command Launcher</em> plugin and integrated with the <em>Script Security</em> plugin.
+      references:
+        - issue: 47393
+        - url: https://jenkins.io/security/advisory/2017-10-11/#arbitrary-shell-command-execution-on-master-by-users-with-agent-related-permissions
+          title: related security advisory
+    - type: rfe
+      issue: 36282
+      pull: 3081
+      message: >
+        Add link to recursive <code>cc.xml</code> output on build history page.
+    - type: major bug
+      issue: 47448
+      pull: 3093
+      message: >
+        Fix <em>Download from java.sun.com</em> installation method for JDK for downloads requiring an Oracle login.
+    - type: bug
+      pull: 3097
+      issue: 47500
+      message: >
+        <code>Secret</code> threw <code>ArrayIndexOutOfBoundsException</code> trying to decrypt <code>{}</code>.
+    - type: bug
+      issue: 47455
+      pull: 3090
+      message: >
+        Race conditions in agents going offline could result in an exception when picking a workspace for a build.
+    - type: bug
+      issue: 35459
+      pull: 2994 # and 3089
+      message: >
+        Prevent duplicated elements with incorrect URL when using the search on Dashboard View plugin based views.
+    - type: bug
+      pull: 3099
+      issue: 47517
+      message: >
+        <code>StackOverflowError</code> thrown under some conditions when using Pipeline on 2.85.
+    - type: bug
+      issue: 47416
+      pull: 3088
+      message: >
+        Prevent <code>NullPointerException</code> updating a folder with a primary view specified in Folders plugin 6.2.0.
+    - type: rfe
+      pull: 3077
+      message: >
+        Developer: Add an empty default implementation for previously <code>abstract</code> methods of <code>SecurityListener</code>.
+    - type: rfe
+      pull: 3091
+      message: >
+        Developer: Deprecate <code>hudson.util.Memoizer</code> and replace its usage in core.
+    - type: bug
+      pull: 3069
+      message: >
+        Developer: <code>Slave.JnlpJar.getURL</code> did not work in some modes when core had a snapshot dependency on <code>remoting</code>.
+    # pull: 3092 too minor (fix README typos)
+    # pull: 3068 too minor (Chinese l10n)
+    # pull: 3085 too minor (Chinese l10n)
```

Comment:
Maybe makes sense to reference the Chinese localization changes. But OTOH it may be preferable to have a single entry once @LinuxSuRen reaches his target milestone

Reply:
I was debating this with myself, and in the end decided not to include it as the changelog is already long enough, and what got merged into this release is an overall rather minor change affecting relatively few core strings (as opposed to e.g. the Bulgarian or German updates in the last year or so, each affecting 200+ files).

Reply:
That would be a long-term job, and I'am going to continue to localize.

Reply:
@LinuxSuRen 👍 

I think this is great, but also that it would spam the changelog to include it every time we merge a contribution of yours 🤷‍♂️